lib/install.js

```diff
@@ -272,8 +272,10 @@ Installer.prototype.run = function (cb) {
         [this, this.saveToDependencies])
     }
   }
-  postInstallSteps.push(
-    [this, this.printInstalled])
+  if (npm.config.get('loglevel') !== 'silent') {
```

Comment:
@deckar01 By those you mean `--json` and `--parseable`? Those _have_ to go to stdout as their output is intended for consumption by other programs. Or do you mean the archy tree that shows up without those flags? I can see the argument that it should go to the log…